content/sensu-core/1.5/guides/pre-compile-plugins.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Pre-Compiling Plugins"
+description: "Pre-Compiling Sensu Plugins for Distribution"
+product: "Sensu Core"
+version: "1.5"
+weight: 2
+next: ../intro-to-checks
+previous: ../overview
```

Comment:
We'll need to remove lines 7 and 8 and change the weight to `12` for all versions

Reply:
removed and changed weight